CoreFoundation/CMakeLists.txt

```diff
@@ -422,6 +426,9 @@ if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
                              PRIVATE
                                ${LIBXML2_INCLUDE_DIR})
   find_package(CURL REQUIRED)
+  target_include_directories(CoreFoundation
```

Comment:
CoreFoundation is not supposed to import or link with libcurl.

Reply:
Yeah, this is based on an older state of the tree.  I will have to redo the work, this was to demonstrate what the build would look like.